src/RuleSet.php

```diff
@@ -286,13 +295,6 @@ final class RuleSet implements RuleSetInterface
             '@Symfony:risky' => true,
             'comment_to_phpdoc' => true,
             'final_internal_class' => true,
-            'function_to_constant' => ['functions' => [
```

Comment:
This removal should be backported to 2.15.

Reply:
I think we should only update the rule sets on the next minor?

Reply:
This is kind of a fix of the ruleset, it can be released in a patch version.